sharktank_models/action.yml

```diff
@@ -0,0 +1,59 @@
+# Copyright 2025 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: IREE Test Suite Regression Test
```

Comment:
Probably want "sharktank models" in this name too. It might have also been easier to start with one of the other test suites since I think we'll want all of them to provide composite actions too.

On that note, can you tag https://github.com/iree-org/iree-test-suites/issues/26 in the PR description?

Like: 

```
Progress on https://github.com/iree-org/iree-test-suites/issues/26
```